AdobeStockImageAdminUi/Block/Adminhtml/Panel.php

```diff
@@ -71,4 +71,14 @@ public function getUserQuota(): UserQuotaInterface
             return $userQuota;
         }
     }
+
+    /**
+     * Get URL for buying more credits
+     *
+     * @return string
+     */
+    public function getBuyCreditsUrl(): string
```

Comment:
I wasn't sure where to put this value

Reply:
I think the best place to put it is `AdobeStockAsset/etc/config.xml` (looks like that will require the creation of Config.php and ConfigInterface for AdobeStockAsset/AdobeStockAssetApi module)